src/transformers/__init__.py

```diff
@@ -609,6 +609,16 @@
         "SpecialTokensMixin",
         "TokenSpan",
     ],
+    "tools": [
+        "Agent",
+        "HfAgent",
+        "OpenAiAgent",
+        "PipelineTool",
+        "RemoteTool",
+        "Tool",
```

Comment:
```suggestion
```

To limit breaking / design changes maybe it might be good to not put too much in the public API?

Reply:
Same